blocks/model.py

```diff
@@ -156,7 +157,11 @@ def __init__(self, outputs):
             if brick not in children and brick not in self.top_bricks:
                 self.top_bricks.append(brick)
         if len(set(b.name for b in self.top_bricks)) < len(self.top_bricks):
-            raise ValueError("top bricks with the same name")
+            # find the first occurences and return it in the error message
+            liste_name = [b.name for b in self.top_bricks]
+            occ = liste_name[numpy.argmax([liste_name.count(liste_name[i])
+                            for i in xrange(len(liste_name))])]
```

Comment:
- liste_name -> list_of_names or name_list

- occ -> repeated_name

- the counting should be much easier to implement with [collections.Counter](https://docs.python.org/2/library/collections.html). You should be interested in its `most_common` method. Also it can be used instead of 
` len(set(b.name for b in self.top_bricks)) < len(self.top_bricks):` with the advantage that it gives you the name of the duplicate.

- the space that you put in the error message after the `same name` and before colon is a little bit non-standard. I think it's better to have write `same name:`